modules/transport-netty4/build.gradle

```diff
@@ -42,6 +42,7 @@ dependencies {
   api "io.netty:netty-handler:${versions.netty}"
   api "io.netty:netty-resolver:${versions.netty}"
   api "io.netty:netty-transport:${versions.netty}"
+  api "io.netty:netty-transport-native-unix-common:${versions.netty}"
```

Comment:
`netty-handler` now has this dependency so we need to add it as well because `SslHandler` does an `instanceof` check using the `UnixChannel` class now.